core/src/main/java/io/kestra/core/models/executions/LogEntry.java

```diff
@@ -128,4 +128,27 @@ public Map<String, String> toMap() {
             .filter(e -> e.getValue() != null)
             .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
     }
+
+    public String toJson(){
```

Comment:
I'm not sure where it is used but it would be better to use an ObjectMapper for that, it would be easier to maintain